readme.md

```diff
@@ -116,7 +116,7 @@ try going to the local host of that port number. In the browser enter
 In the browser we'll see something like :
 
 ```
-CANNOT GET /
+cannot GET /
```

Comment:
Cannot *

![image](https://cloud.githubusercontent.com/assets/8976305/20006708/39d7c412-a26f-11e6-82e7-4a2a9a63f921.png)

not a big deal